src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,68 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
+import { Options } from '../../types/Options';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  onSelectedOption: (selectedOption: Options) => void;
+  onInputChange: (query: string) => void;
+  inputValue: string,
+  onClearQuery: ()=> void,
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+const SELECTED_OPTIONS = [Options.ALL, Options.ACTIVE, Options.COMPLETED];
+
+export const TodoFilter: React.FC<Props> = ({
+  onSelectedOption,
+  onInputChange,
+  inputValue,
+  onClearQuery,
+}) => {
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            data-cy="statusSelect"
+            onChange={e => onSelectedOption(e.target.value as Options)}
+          >
+            {SELECTED_OPTIONS.map(option => {
+              return (
+                <option
+                  value={option.toLowerCase()}
```

Comment:
It`s worse idea, because the letter in naming of select options in both cases. In other way I have to write a function to name it properly, because test accepts it only in that way